addons/web_editor/static/lib/odoo-editor/test/spec/format.test.js

```diff
@@ -141,7 +141,13 @@ describe('Format', () => {
             await testEditor(BasicEditor, {
                 contentBefore: `<p>a${spanBold(`b${strong(`c${b(`[d]`)}e`)}f`)}g</p>`,
                 stepFunction: bold,
-                contentAfter: `<p>a${spanBold(`bc`)}[d]${spanBold(`ef`)}g</p>`,
+                contentAfter: `<p>
+                    a
+                    <span style="font-weight: bolder;">b<strong>c</strong></span>
+                    [d]
+                    <span style="font-weight: bolder;"><strong>e</strong>f</span>
+                    g
+                </p>`.replaceAll(/\n\s*/g, ''),
```

Comment:
I think it is very complex to read the test if the `contentBefore` and `contentAfter` uses different ways to express the html. Either use the magic `${...}` notation in both of them, or fully write the html of both of them. If you choose to write both of them fully, then I suggest to use the `unformat` util which does the removal of the indentation whitespaces.